src/main/java/org/folio/des/controller/JobsController.java

```diff
@@ -50,6 +53,15 @@ public ResponseEntity<Job> upsertJob(@RequestHeader("X-Okapi-Tenant") String ten
     return new ResponseEntity<>(service.upsertAndSendToKafka(job, true), job.getId() == null ? HttpStatus.CREATED : HttpStatus.OK);
   }
 
+  @Override
+  public ResponseEntity<Resource> downloadExportedFileByJobId(UUID id) {
+    try {
+      return ResponseEntity.ok(new InputStreamResource(service.downloadExportedFile(id)));
+    } catch (Exception e) {
+      throw new RuntimeException(e.getMessage());
```

Comment:
If you use new runtime exception FileDownloadException - this catch block will be not needed, and this new exception will be handled in appropriate Exception Controller Advice class